### ISSUE TITLE
Entitas Forum

### ISSUE THREAD

**Issue by sschmid** [sschmid/Entitas-CSharp#365]
## Hello Community!

I'm happy to see that more and more people are using Entitas. Exciting times to be an Entitas dev at the moment, Roslyn Code Generation is around the corner which will fix the code generation issue when the code is not compiling. Entitas-Lang, our own DSL, which lets you write concise component and system definitions just got pushed open source. I started a full step-by-step tutorial which explains the Entitas vision as well as all the concepts.

I'm also happy to see active and motivated people in the chat and that you guys help each other whenever you can. We now reached a size where it definitely makes sense to become more organized since the chat might be hard to follow sometimes.

This issue kicks off the search for a great forum. I think having an Entitas forum will help people getting started and to encourage more discussions. There can be starter guides, advanced topics, best practices and a show-off section where you can present your game #madeWithEntitas.

I don't have strict requirements for a forum, but it should be modern, nice to use, easy to search, shouldn't look like it's straight from the ´80s and in best case it should be free.

If you know forums that you like, please post them here and tell me why you think this forum is great and why it should be used. If you have further requirements to a forum you can also post them here. This way we hopefully find a great solution we're all happy with.

**Comment by WoLfulus** [comment]
if you don't mind helping newer projects and using beta stuff, this is amazing:
http://flarum.org/

**Comment by buihuuloc** [comment]
Also, How about this platform https://www.discourse.org/?

**Comment by cloudjubei** [comment]
discourse is paid?

**Comment by WoLfulus** [comment]
@cloudjubei if you pay them to host it then yes

**Comment by sschmid** [comment]
We reached 1000 starts today, that should be enough motivation to finally start a forum :D
I'd love to go with flarum, but they emphasize not using it in production.
Probably will take a look at discourse then

**Comment by JamesMcMahon** [comment]
https://blog.discourse.org/2016/03/free-discourse-forum-hosting-for-community-friendly-github-projects/

> Free Discourse Hosting For Your Open Source Project
Do you run a popular open source project on GitHub? Then you might be eligible for a free, hosted Discourse + SSL. Your users can log in with GitHub logins and write posts in Markdown, and we’ve got tighter integration planned down the road, too.
Minimum Requirements
Must be hosted on GitHub
2000+ stars
30+ contributors
Your project has a clear need for discussions (see FAQ below)
Please understand that approval is ultimately at our discretion as we work out the details of this program.

**Comment by sschmid** [comment]
### Ok guys, make it rain stars then! :)
⭐️ ⭐️ ⭐️ ⭐️ ⭐️

**Comment by kewlniss** [comment]
Would a dedicated forum be better than simply using StackOverflow with an Entitas tag? All of the awesome folks in the chat could monitor SO for questions as they had time. And I believe SO's SEO juice would be better than any particular forum. Perhaps this was actively decided against, but I figured I'd ask since I've seen other projects like this do well by simply tagging questions in StackOverflow.
